Header header #Header for message type
float64 robot_x #robot front movement Basic movement commands
float64 robot_y #robot side movement
float64 robot_zrot #robot z rotation
float64 body_left #robot body track left
float64 body_right #robot body track right
float64 arm_front_left # angle
float64 arm_front_right # angle
float64 arm_back_left # angle
float64 arm_back_right # angle
float64 mecanum_front_left # velosity
float64 mecanum_front_right # velosity
float64 mecanum_back_left # velosity
float64 mecanum_back_right # velosity
